spec/support/shared_contexts/api/docs/api_update_endpoint_documentation_support.rb

```diff
@@ -0,0 +1,65 @@
+RSpec.shared_context "an API update endpoint documentation", :exceptions_app do |options = {}|
+  path options[:url] do
+    put "Updates a #{options[:resource_description]}" do
```

Comment:
Hi @ethax-ross for consistency.. should we use 3rd person singular here or as just as the other endpoints that use "Retrieve" instead?